pgcli/main.py

```diff
@@ -819,6 +830,7 @@ def get_last_query(self):
         default=False,
         help='Skip intro on startup and goodbye on exit.')
 @click.option('--prompt', help='Prompt format (Default: "\\u@\\h:\\d> ").')
+@click.option('--prompt-dsn', help='Prompt format when DSN alias is used (Default: "\\dsn> ").')
```

Comment:
You named it `'\\dsn_alias'` elsewhere, but `'\\dsn'` here.

Also, I don't see you actually setting the default.

Reply:
Yep, it's artifacts from previous version, sorry. About default: yep, it's just using default from "normal" prompt.
Fixed now :)